src/yb/master/backfill_index.cc

```diff
@@ -678,6 +667,21 @@ BackfillTable::BackfillTable(
   done_.store(false, std::memory_order_release);
 }
 
+void BackfillTable::retrieve_index_names(const std::unordered_set<std::string>& requested_index_ids, std::string& requested_id_names) {
```

Comment:
Our internal coding style is largely based on https://google.github.io/styleguide/cppguide.html#Function_Names 

I'd suggest using CamelCase to name the function RetrieveIndexNames or IndexNamesForIndexIds(...)

Also, it would be preferable to return the string; instead of using an output parameter